builtin/settingtypes.txt

```diff
@@ -418,7 +418,10 @@ enable_particles (Digging particles) bool true
 
 [**Filtering]
 
-#    Use mip mapping to scale textures. May slightly increase performance.
+#    Use mip mapping to scale textures. May slightly increase performance,
+#    especially when using a high resolution texture pack.
+#    Note that it currently lacks gamma correct downscaling generating the mip
+#    maps, thus the textures further away look a bit darker.
```

Comment:
*darker*? How can you be sure they can’t look *lighter* with some monitor settings?
Seriously, that doesn't belong to docs. These tiny imperfections don’t worth it. (BTW, remember that in-game “gamma” is applied to light levels and not to resulting pixels, that may be much more noticeable).

Reply:
I still think missing gamma correction should be mentioned, but using only very few words.

> These tiny imperfections don’t worth it.

The brick texture mipmaps look quite different with gamma correction. (images from #6917)
default:
![d](https://user-images.githubusercontent.com/3192173/35196439-5321918a-fed2-11e7-8d2b-9f5c643fad19.png)

gamma-correct:
![g](https://user-images.githubusercontent.com/3192173/35226318-38420264-ff8b-11e7-8ccd-2460b5422e8e.png)
(you can ignore the bilinear or so filter)

Reply:
@HybridDog What you post as “gamma-corret,” in fact, looks *worse* on my display.